packages/admin-ui/src/radio/radio-group.style.ts

```diff
@@ -18,13 +18,13 @@ export const variants = styleVariants({
     vertical: {
       flexDirection: 'column',
       '> :not(:last-child)': {
-        marginBottom: '$l',
+        marginBottom: '$space-2',
       },
     },
     horizontal: {
       flexDirection: 'row',
       '> :not(:last-child)': {
-        marginRight: '$xl',
+        marginRight: '$space-5',
```

Comment:
This value is correct, but there's an issue (maybe with the inline component) that's making the radio have 8px of margin before the circle.